invenio_requests/records/jsonschemas/request_events/event-v1.0.0.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "local://request_events/event-v1.0.0.json",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
+    },
+    "id": {
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "request_id": {
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "type": {
+      "type": "string"
+    },
+    "extras": {
+      "type": "object",
```

This puts a bit harder structure on the keys.
```suggestion
      "$ref": "local://requests/definitions-v1.0.0.json#/anything"
```